WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -601,45 +572,33 @@ private extension StoreStatsV4PeriodViewController {
     func reloadAllFields(animateChart: Bool = true) {
         viewModel.selectedIntervalIndex = nil
         reloadSiteVisitUI()
-        updateConversionStatsVisibility(visitStatsMode: siteVisitStatsMode)
         reloadChart(animateChart: animateChart)
-        let visitStatsElements: [Any] = {
-            switch siteVisitStatsMode {
-            case .default:
-                return [visitorsTitle as Any,
-                        visitorsData as Any]
-            case .redactedDueToJetpack:
-                return [visitorsTitle as Any,
-                        visitorsEmptyView as Any]
-            case .hidden:
-                return []
-            }
-        }()
 
-        view.accessibilityElements = visitStatsElements + [ordersTitle as Any,
-                                                           ordersData as Any,
-                                                           revenueTitle as Any,
-                                                           revenueData as Any,
-                                                           conversionTitle as Any,
-                                                           conversionData as Any,
-                                                           yAxisAccessibilityView as Any,
-                                                           xAxisAccessibilityView as Any,
-                                                           chartAccessibilityView as Any]
+        view.accessibilityElements = [ordersTitle as Any,
+                                      ordersDataOrRedactedView as Any,
+                                      visitorsTitle as Any,
+                                      visitorsDataOrRedactedView as Any,
+                                      revenueTitle as Any,
+                                      revenueData as Any,
+                                      conversionTitle as Any,
+                                      conversionDataOrRedactedView as Any,
+                                      yAxisAccessibilityView as Any,
+                                      xAxisAccessibilityView as Any,
+                                      chartAccessibilityView as Any]
     }
 
     func reloadSiteVisitUI() {
+        guard visitorsDataOrRedactedView != nil else {
+            return
+        }
```

Comment:
Again, do we really need this check here?

Reply:
Since not all views were checked in the old diffs, I removed the check here in https://github.com/woocommerce/woocommerce-ios/pull/5932/commits/95bc32a8a59ba1f3478318d6fe13bc3288725868